tests/fixture/wrangler_toml.rs

```diff
@@ -95,6 +156,7 @@ impl WranglerToml<'_> {
         wrangler_toml.workers_dev = Some(true);
         wrangler_toml.target_type = Some("javascript");
 
+        eprintln!("{:#?}", &wrangler_toml);
```

Comment:
these are handy for when your test fails and you want to see what the test toml looked like

Reply:
i killed these later because they get obnoxious when you compose them like ive been doing.

Reply:
these two comments are way at the bottom of the review process :p i've added a bunch of comments about removing them but if you do it in a subsequent PR that's fine by me